src/vs/workbench/contrib/audioCues/browser/audioCueService.ts

```diff
@@ -219,10 +220,16 @@ export class AudioCue {
 		settingsKey: 'audioCues.noInlayHints'
 	});
 
-	public static readonly taskEnded = AudioCue.register({
-		name: localize('audioCues.taskEnded', 'Task Ended'),
-		sound: Sound.taskEnded,
-		settingsKey: 'audioCues.taskEnded'
+	public static readonly taskCompleted = AudioCue.register({
+		name: localize('audioCues.taskCompleted', 'Task Completed'),
```

Comment:
Task success instead of completed? Failed may have completed as well

Reply:
if this is going to play when a build completes too (background watch tasks), build makes more sense